.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "none",
```

Comment:
```suggestion
  "trailingComma": "all",
```

Reply:
sure - as mentioned, I rarely care about differing opinions on spacing, everyone has them, I just want a standard that is auto-enforced, as a repo maintainer so that all review attention can focus on content vs style